#!/usr/bin/python3
def raise_exception():
    raise TypeError("This is a deliberate TypeError")
